### PR TITLE
CMR-4751 Add three mappings to translate ECHO 10 Contact Contacts <=> UMM-C

### PR DESCRIPTION
**Pull request by daniel-zamora**

### CODE REVIEW

File: umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/echo10/data_contact.clj [hunk 1]
```diff
@@ -45,7 +46,9 @@
   "Science Software Development Manager" "Science Software Development"
   "Deputy Science Software Development Manager" "Science Software Development"
   "Sea Ice Algorithms" "Science Software Development"
-  "Snow Algorithms" "Science Software Development"})
+  "Snow Algorithms" "Science Software Development"
+  "Science Contact" "Science Contact"
+  "Science Software Development" "Science Software Development"})
```
Comment (sxu123) [inline umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/echo10/data_contact.clj]: In the ticket, it asked "Also please make the ECHO 10 map values case insensitive."  I don't see any case insensitive code?

File: umm-spec-lib/src/cmr/umm_spec/umm_to_xml_mappings/echo10/data_contact.clj [hunk 2]
```diff
@@ -16,7 +16,7 @@
   "Investigator" "INVESTIGATOR"
   "Metadata Author" "METADATA AUTHOR"
   "User Services" "User Services"
-  "Science Software Development" "Science Software Development Manager"})
+  "Science Software Development" "Science Software Development"})
```
Comment (sxu123) [inline umm-spec-lib/src/cmr/umm_spec/umm_to_xml_mappings/echo10/data_contact.clj]: In the ticket, it asked for "Technical Contact for Science to Science Contact"  Does it no longer apply? Have you clarify with @eereiter?

Comment (sxu123) [reply umm-spec-lib/src/cmr/umm_spec/umm_to_xml_mappings/echo10/data_contact.clj]: Actually I see it "Science Contact" "Technical Contact for Science" on line 14, Should change the value to "Science Contact"?

File: umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/echo10/data_contact.clj (hunk 1 above)
Comment (daniel-zamora) [reply umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/echo10/data_contact.clj]: Current Acceptance Criteria doesn't include that.
Ingest the attached echo 10 record.
Retrieve the same record using the umm-json specification.
Verify that the Data Contact named "National Snow and Ice Data Center Originator" has the role of ORIGINATOR
Verify that the Data Contact Contact Person has a role of Science Contact and the other has a role of Science Software Development
Ingest the retrieved umm-json record and retrieve it as an echo 10 format record. Verify that a contact exits with two contact persons. The contacts role is ORIGINATOR and one contact person has a JobPosition of Science Contact and the other has a JobPosition of
Science Software Development.

File: umm-spec-lib/src/cmr/umm_spec/umm_to_xml_mappings/echo10/data_contact.clj (hunk 2 above)
Comment (daniel-zamora) [reply umm-spec-lib/src/cmr/umm_spec/umm_to_xml_mappings/echo10/data_contact.clj]: done

**Pull request merged**